backend/src/main/java/corea/participation/domain/Participation.java

```diff
@@ -82,6 +82,10 @@ public boolean isReviewer() {
         return memberRole.isReviewer();
     }
 
+
+    public boolean isNotReviewer() {
+        return !isReviewer();
+
```

Comment:
```suggestion
        return !isReviewer();
}
```

conflict 수정하면서 중괄호 빠짐!